modules/claims_api/lib/claims_api/vbms_sidekiq.rb

```diff
@@ -6,6 +6,20 @@ module ClaimsApi
   module VBMSSidekiq
     include SentryLogging
 
```

Comment:
Just making doubly-sure, but all this change did was move the existing logic closer to the top of the class, right?
That's what it looks like and I love the change BTW.  Just want to make sure I'm not overlooking some additional change.